plugins/events-backend-module-kafka/src/index.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2025 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * The module "kafka" for the Backstage backend plugin "events"
+ * adding an Kafka-based publisher,
+ * receiving events from an Kafka topic and passing it to the
+ * internal event broker.
+ *
+ * @packageDocumentation
+ */
+
+export { KafkaConsumerClient } from './publisher/KafkaConsumerClient';
+export { KafkaConsumingEventPublisher } from './publisher/KafkaConsumingEventPublisher';
+export { eventsModuleKafkaConsumingEventPublisher as default } from './service/eventsModuleKafkaConsumingEventPublisher';
+export type {
+  KafkaConsumerConfig,
+  KafkaEventSourceConfig,
+} from './publisher/config';
```

Comment:
```suggestion
export { eventsModuleKafkaConsumingEventPublisher as default } from './service/eventsModuleKafkaConsumingEventPublisher';
```

I think you only need to expose the module and nothing else. This will reduce the API surface a lot, too.